blog/2024-05-20-standing-up-a-fhir-gateway-for-dhis2-from-an-ig/index.md

```diff
@@ -0,0 +1,553 @@
+---
+slug: 2024/05/standing-up-a-fhir-gateway-for-dhis2-from-an-ig
+title: Standing up a FHIR Gateway for DHIS2 from an Implementation Guide
+authors: claudemamo
+tags: [fhir, camel, ig, java]
+---
+
+A modern vision of health interoperability is one where systems exchange health care data using the [FHIR (Fast Health Interoperability Resources)](https://en.wikipedia.org/wiki/Fast_Healthcare_Interoperability_Resources) standard to describe the types of resources and data to be exchanged. FHIR outlines these resources and data using what is called an [Implementation Guide (IG)](https://www.hl7.org/fhir/implementationguide.html). An IG specifies the rules for interoperability in a formal, machine-readable, and testable way. In this post, we will show you how to stand up a FHIR gateway for DHIS2 from an IG so that DHIS2 can interoperate with FHIR clients.
```

Comment:
```suggestion
A modern vision of health interoperability is one where systems exchange health care data using the [FHIR (Fast Health Interoperability Resources)](https://en.wikipedia.org/wiki/Fast_Healthcare_Interoperability_Resources) standard to describe the types of resources and data to be exchanged. FHIR outlines these resources and data using what is called an [Implementation Guide (IG)](https://www.hl7.org/fhir/implementationguide.html). An IG specifies the rules for interoperability in a formal, machine-readable, and testable way. In this post, we will show you how to set up a FHIR gateway for DHIS2 from an IG so that DHIS2 can interoperate with FHIR clients.
```